source/payment_flow/index.html.md.erb

```diff
@@ -208,6 +208,11 @@ will then:
 
 Find out [when you'll receive the payment](/integrate_with_govuk_pay/#when-you-39-ll-receive-payments).
 
+You can change what appears on your user's bank statement by:
+
+- [contacting us](/support_contact_and_more_information/#support) - if your PSP is Stripe
```

Comment:
Looks good to me - one thing is maybe it should be: - [contacting us](/support_contact_and_more_information/#support) (if your PSP is Stripe) ... so having "if your PSP is Stripe" in brackets. A dash looks like there's more longer information to be read rather than just an aside bit of information, if that makes sense

Reply:
Amended in https://github.com/alphagov/pay-tech-docs/pull/395/commits/0352ef03a2e7dcc0561d31071b8b41f324b32062.